### ISSUE TITLE
Do not mask comments in the migration datamodel files

### ISSUE THREAD

**Issue by lasharela** [prisma/migrate#234]
```
datasource db {
  provider = "postgresql"
  // url      = "postgresql://@#$@localhost/prisma2"
  url      =  "real connection string was here"
}
```

As you see one of the url is commented, but in a console after `prisma migrate up` i see:

```
datasource db {
  provider = "postgresql"
     url = "***"
  url = "***"
}
```
both URL's

**Comment by pantharshit00** [comment]
I think we should remove comments from our credentials masking system.

**Comment by janpio** [comment]
> but in a console after prisma2 lift up i see

What location does this refer to @lasharela exactly?

> I think we should remove comments from our credentials masking system.

What do you mean exactly @pantharshit00? Could you please update the issue title to be more descriptive?

**Comment by pantharshit00** [comment]
We use the following function to mask our our schema so that we don't expose credentials in the migrations files: https://github.com/prisma/lift/blob/0c8570d6aa4c199c3f1786ad7daa2041d0567aa2/src/utils/maskSchema.ts

The regexp that it is using doesn't account for comments. It would mask the url regardless it is a valid line or it is a function. So we need to include changes such that it accounts for comments.

**Comment by Jolg42** [comment]
Result should look like 
```
datasource db {
  provider = "postgresql"
  // url = "***"
  url = "***"
}

```

**Comment by tomhoule** [comment]
Assigning @Jolg42 to confirm that this is fixed.

**Comment by tomhoule** [comment]
Thanks for taking the time to report this issue!

We've recently released a Prisma Migrate Preview ([2.1.3.0 release notes](https://github.com/prisma/prisma/releases/tag/2.13.0)), which has quite a few changes compared to the previous experimental version. We believe this issue is no longer relevant in this new version, so we are closing this.

We would encourage you to try out Prisma Migrate Preview. If you come across this or any other issue in the preview version, please feel free to open a [new issue in prisma/prisma](https://github.com/prisma/prisma/issues/new/choose).
For general feedback on Prisma Migrate Preview, feel free to chime in [on this issue](https://github.com/prisma/prisma/issues/4531).
